src/app/components/applications/SSV/RegisterValidatorHome/components/ImportFile/flows/KeyShares.tsx

```diff
@@ -154,15 +159,16 @@ const KeyShareFlow = () => {
     if (!shares.length) {
       return getResponse(KeyShareValidationResponseId.OK_RESPONSE_ID);
     }
-    if (shares.length > 1) {
+    if (shares.length > 1 && getBulkMode() === EBulkMode.MULTI) {
```

Comment:
I think it will be a bit cleaner to add a isBulkMode(mode: EBulkMode): boolean { return getBulkMode() === mode } to useValidatorRegistrationFlow.ts

Reply:
V